Capstone MeetApp/apps/meet-app-api/src/app/app.module.ts

```diff
@@ -1,10 +1,11 @@
 import { Module } from '@nestjs/common';
-
+import { ApiBackendModule } from '../../../../libs/api/backend/src'
```

Comment:
I think there is a different way for defining the path checkout tsconfig.base.json ...use the @capstone ... path